garden-share-08/garden-share-fe
# Feature cookies

#### What does  this PR do?
- Adds Cookies to store userId
- Completes api queries for listings and offers
- Renders confirmation for a new offer
- Removes declined offers from DOM

#### Where should the reviewer start?
- actions.js
- MyListings, MyOffers, UserListing, UserOffer
- ProductPage
- App
- HomePage

#### How should this be manually tested?
- Navigate to a page other than Homepage and refresh.  The page should persist.
- In Profile, you can view, accept, or decline offers.

#### Any background context you want to provide?
- This branch DOES NOT include the change to cookies needed for logout.  Upon logout, the userId should be REMOVED from the cookies.  That is a next step.

#### What are the relevant tickets?
- closes #37 
- closes #42 
- closes #46 
- closes #51 
- closes #52
